tests/ast/test_ast_driver.py

```diff
@@ -277,6 +279,9 @@ def test_ast(session, test_case):
         session, test_case.filename.replace(".", "_"), test_case.source
     )
     if pytest.update_expectations:
+        assert (
+            pytest.unparser_jar
+        ), "Can only update expectations with unparser jar set."
```

Comment:
nit: add actionable inside for user, i.e. some hint on what to do in order to fix it.

Reply:
done! is the new message good or should I include more details?